modules/http/src/main/java/org/mule/extension/http/internal/request/ConfigurationOverrides.java

```diff
@@ -21,22 +21,6 @@
 public final class ConfigurationOverrides {
 
   /**
-   * Host where the requests will be sent.
```

Comment:
The use case for overriding host and port is kind of obsolete to me when you can override the entire url.